lms/views/lti/deep_linking.py

```diff
@@ -123,7 +123,13 @@ def file_picker_to_form_fields_v13(self):
             "https://purl.imsglobal.org/spec/lti/claim/message_type": "LtiDeepLinkingResponse",
             "https://purl.imsglobal.org/spec/lti/claim/version": "1.3.0",
             "https://purl.imsglobal.org/spec/lti-dl/claim/content_items": [
-                {"type": "ltiResourceLink", "url": document_url}
+                {
+                    "type": "ltiResourceLink",
+                    "url": self.misc_plugin.get_deeplinking_launch_url(
+                        self.request, assignment_configuration
+                    ),
+                    "custom": assignment_configuration,
```

Comment:
Send the configuration for all DL assignments. We'll only use it D2L now but no need to make it D2L specific and bring a plugin method here.

This is part of the spec https://www.imsglobal.org/spec/lti-dl/v2p0#deep-linking-response-example